src/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.cpp

```diff
@@ -119,7 +119,7 @@ tnsr::a<DataType, SpatialDim, Frame> gauge_source_function(
   for (size_t i = 0; i < SpatialDim; ++i) {
     gauge_source_h.get(i + 1) +=
         one_over_lapse *
-        (deriv_lapse.get(i) - trace_christoffel_last_indices.get(i));
+        deriv_lapse.get(i) - trace_christoffel_last_indices.get(i);
```

Comment:
This changes the math happening here, I just want to explicitly check you meant to do it :)

Reply:
Yes I did...previous was a bug.

Reply:
Okay, sounds good :+1:

Reply:
The factor of lapse was not supposed to be in front of trace_christoffel_last_indices.